disttask/framework/dispatcher/dispatcher.go

```diff
@@ -374,6 +382,30 @@ func GenerateSchedulerNodes(ctx context.Context) ([]*infosync.ServerInfo, error)
 	return serverNodes, nil
 }
 
+func (d *dispatcher) filterByRole(infos []*infosync.ServerInfo) ([]*infosync.ServerInfo, error) {
+	nodes, err := d.taskMgr.GetNodesByRole("background")
```

Comment:
Will label case sensetive?

Reply:
No, when we set it, we will convert it to lower case.